kotsadm/pkg/handlers/snapshots.go

```diff
@@ -525,3 +529,27 @@ func parseTTL(s string) (*snapshottypes.ParsedTTL, error) {
 	}
 	return parsedTTLResponse, nil
 }
+
+func CheckIsVeleroInstalled(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Access-Control-Allow-Origin", "*")
+	w.Header().Set("Access-Control-Allow-Headers", "content-type, origin, accept, authorization")
+
+	if r.Method == "OPTIONS" {
+		w.WriteHeader(http.StatusOK)
+		return
+	}
+
+	getIsVeleroInstalledResponse := VeleroInstalled{}
+
+	detectVelero, err := snapshot.DetectVelero()
+	if err != nil {
+		logger.Error(err)
```

Comment:
Can we get this error returned to the UI maybe?  Looks like there is error handling in the UI, but this handler always returns 200.